src/ray/raylet/worker_pool.cc

```diff
@@ -151,19 +151,27 @@ void WorkerPool::SetAgentManager(std::shared_ptr<AgentManager> agent_manager) {
   agent_manager_ = agent_manager;
 }
 
+void WorkerPool::PopWorkerCallbackAsync(const PopWorkerCallback callback,
```

Comment:
I think this helper method is unnecessary. I saw that some callback invocations do not leverage this method.